submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,382 @@
+
+Based on these examples, it seems that while modules do offer some significant boosts to energy gains, they do not necessarily make every single fuel processing step offer net gains.
+
+## So how useful is fuel processing in general?
+
+Our findings so far about energy gains only partially help us answer this question. I have identified a more comprehensive list of factors that you should consider when deciding whether to (further) process your burner fuels:
+
+* **Gains in energy:** If you get more energy out of the processed fuel, then your demand for the unprocessed fuel decreases and so your resource patches run out more slowly. We determined the energy gains for different fuels earlier.
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses as seen in [the wiki page for fuels](https://wiki.factorio.com/Fuel).
+- **Gains in energy density (per unit and per stack):**  More energy per stack means the same inventory slot can hold more energy, which makes a significant difference for how much energy you can fit into each cargo wagon or chest. Meanwhile, more energy per unit means more energy can be carried by robots, belts, and inserters, which can only hold a few units at a time. If your energy consumption is fixed, using a more energy-dense fuel means needing less logistics infrastructure to move fuels, which means simpler designs and less energy consumed by the logistic system itself. More processed fuels have greater energy density in both regards, hence having these benefits.
+- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock costs of the required technologies. Each tier of processing adds to these costs.
+- **Availability of more coal/oil to just use instead:** On most maps, coal and oil are practically infinite, but you may not have any new patches nearby. Depending on your preferences, it may be easier to head out and set up more resource extraction infrastructure rather than building more fuel processing infrastructure.
```

Comment:
```suggestion
- **Availability of more coal/oil to just use instead:** On most maps, coal and oil are practically infinite, but you may not have any new patches nearby. Depending on your preferences, it may be easier to head out and set up more resource extraction infrastructure, rather than building more fuel processing infrastructure.
```